kafka/manager_test.go

```diff
@@ -337,11 +356,9 @@ func TestManagerMetrics(t *testing.T) {
 		}},
 	}, metrics[0].Data, metricdatatest.IgnoreTimestamp())
 
-	assert.Equal(t, &kmsg.ListGroupsRequest{Version: 4}, listGroupsRequest)
-	assert.Equal(t, &kmsg.DescribeGroupsRequest{
-		Version: 5,
-		Groups:  []string{"connect", "consumer1", "consumer2", "consumer3"},
-	}, describeGroupsRequest)
+	assert.Nil(t, listGroupsRequest)
```

Comment:
Should we remove the `cluster.ControlKey(kmsg.ListGroups.Int16()...`? I don't think we need to test which APIs we're not calling.